integration-tests/.env

```diff
@@ -0,0 +1,4 @@
+RAW_EMB_URL=
```

Comment:
can you update `integration-tests/README.md` with an example of how the integration tests should now be run with these env vars? The old instructions could be removed as well as `helpers/invoke_embrace_server.ts`